src/arch/xtensa/hal/interrupts.c

```diff
@@ -415,9 +415,9 @@ XtHalVoidFunc *Xthal_tram_trigger_fn = xthal_null_func;
 unsigned xthal_vpri_to_intlevel(unsigned vpri)
 {
```

Comment:
@PayalKshirsagar please don't touch this code. Is taken from xtensa and we are not changing the coding style here.

Reply:
Okay, Thank you:)